packages/techdocs-common/src/stages/prepare/url.ts

```diff
@@ -19,17 +19,32 @@ import { UrlReader } from '@backstage/backend-common';
 import { Entity } from '@backstage/catalog-model';
 import { Logger } from 'winston';
 import { getDocFilesFromRepository } from '../../helpers';
-import { PreparerBase, PreparerResponse } from './types';
+import { PreparerBase, PreparerResponse, UrlFactory } from './types';
 
+/**
+ * Preparer used to retrieve documentation files from a remote repository
+ * @public
+ */
 export class UrlPreparer implements PreparerBase {
   private readonly logger: Logger;
   private readonly reader: UrlReader;
 
+  /**  @deprecated use static fromConfig method instead */
   constructor(reader: UrlReader, logger: Logger) {
     this.logger = logger;
     this.reader = reader;
   }
 
+  /**
+   * Returns a directory preparer instance
+   * @param config - A backstage config
```

Comment:
Are these `@param`s needed (and in particular, I don't think this `config` param is real)

Reply:
You're right, removed!